controllers/actions.github.com/ephemeralrunner_controller.go

```diff
@@ -521,6 +521,14 @@ func (r *EphemeralRunnerReconciler) updateStatusWithRunnerConfig(ctx context.Con
 	jitSettings := &actions.RunnerScaleSetJitRunnerSetting{
 		Name: ephemeralRunner.Name,
 	}
+
+	for c := range ephemeralRunner.Spec.Spec.Containers {
```

Comment:
Can you please rename `c` to `i`? c sounds like a container, while i is usually variable associated with iterations.

Reply:
Yep, thanks, fixed.